From 432aec7aa13b43ad34f635e3cc87a3c20011d3f1 Mon Sep 17 00:00:00 2001
From: OpenStack Release Bot <infra-root@openstack.org>
Date: Tue, 20 Sep 2022 08:57:34 +0000
Subject: [PATCH] Update .gitreview for stable/zed

Change-Id: Iee3c332437de989b67f7476fa09e727472ef641a
---

diff --git a/test-requirements.txt b/test-requirements.txt
index 2a9084b1..14bd96a2 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -12,3 +12,4 @@ testtools>=2.3.0 # MIT
 stestr>=2.0.0 # Apache-2.0
 reno>=3.1.0 # Apache-2.0
 zake>=0.1.6 # Apache-2.0
+WebTest>=3.0.0 # MIT
diff --git a/vitrage/storage/impl_sqlalchemy.py b/vitrage/storage/impl_sqlalchemy.py
index ed049288..c7db25b9 100644
--- a/vitrage/storage/impl_sqlalchemy.py
+++ b/vitrage/storage/impl_sqlalchemy.py
@@ -42,6 +42,7 @@ class Connection(base.Connection):
         for opt in storage.OPTS:
             options.pop(opt.name, None)
         self._engine_facade = db_session.EngineFacade(self._dress_url(url),
+                                                      autocommit=True,
                                                       **options)
         self._active_actions = ActiveActionsConnection(self._engine_facade)
         self._events = EventsConnection(self._engine_facade)
-- 
2.34.1

